packages/shared/lib/core/wallet/actions/claimActivity.ts

```diff
@@ -21,9 +23,18 @@ export async function claimActivity(activity: Activity): Promise<void> {
                 claimingTransactionId: results[0].transactionId,
                 claimedDate: new Date(),
             })
+
+            showAppNotification({
+                type: 'info',
+                message: localize('general.claimedSuccess'),
+            })
         }
     } catch (err) {
         console.error(err)
+        showAppNotification({
+            type: 'error',
+            message: localize('general.claimedFailed'),
```

Comment:
this is different in the en.json file
```suggestion
            message: localize('general.claimedError'),
```